src/SearchInput/SearchInput.tsx

```diff
@@ -42,7 +51,12 @@ export const SearchInput = ({
       sx={{ flex: 1, width: width ?? '100%', backgroundColor: 'white' }}
       placeholder={placeholder}
       size={size}
-      inputProps={{ 'aria-label': ariaLabel }}
+      inputProps={{
+        'aria-label': ariaLabel,
+        // Umami data props
+        'data-umami-event': dataUmami?.event,
+        'data-umami-event-page': dataUmami?.page,
```

Comment:
```suggestion
        // Umami data props
        {...buildUmamiProps(dataUmami)}
```
Not fan of spread props tho, but you're then free to add context if you need later.